addons/external-player-button/addon.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "External player button",
+  "description": "Adds buttons for viewing projects on forkphorus or other sites.",
+  "credits": [
+    {
+      "name": "HTML-Fan/NT"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "options": [
+    {
+      "name": "Add forkphorus",
+      "id": "forkphorus",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Add turbowarp",
+      "id": "turbowarp",
+      "type": "boolean",
+      "default": false
+    },
+    {
+      "name": "Use colors",
+      "id": "colors",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Replace stage",
+      "id": "stage",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "name": "Click again to get to the page",
+      "id": "again",
+      "type": "boolean",
+      "default": false
+    }
+  ],
+  "tags": ["projects"],
+  "enabled_by_default": true
```

Comment:
Just set this to false. And tags to "community"

Reply:
Ah okay

Reply:
Remove "projects".

Reply:
Um doesn't it mess with the project pages?

Reply:
Whatever